app/components/folio/newsletter_subscriptions/form_component.slim

```diff
@@ -0,0 +1,29 @@
+div class=bem_class_name data=data
+  = form do |f|
+    - remember_option_keys.each do |opt|
+      - if @view_options[opt]
+        input[
+          type="hidden"
+          name="view_options[#{opt}]"
+          value=ERB::Util.html_escape(@view_options[opt])
+        ]
+
+    - if @view_options[:input_label]
+      .form-group.m-0
+        label.form-label for="newsletter_subscription[email]"
+          = @view_options[:input_label]
+
+    .position-relative
+      = input(f)
+
+      button.f-newsletter-subscriptions-form__btn[
+        type="submit"
+        class="btn btn-primary"
+      ]
+        = submit_text
+
+    span.folio-loader.f-newsletter-subscriptions-form__loader
+
+  - if @persisted
+    .f-newsletter-subscriptions-form__message
+      = message
```

Comment:
v pripade `@persisted` a tedy `f-newsletter-subscriptions-form--persisted` by mel formular zmizet (pres visibility: hidden) a mela by se objevit zprava; to se ted nedeje